### DISCUSSION TITLE
How TikTok Gave Me the Perfect Coding Side Project – Benjamin Carlson

### DISCUSSION THREAD

**Discussion by giscus** [bjcarlson42/benjamincarlson.io#44] [General]
# How TikTok Gave Me the Perfect Coding Side Project – Benjamin Carlson

I got an idea for a weekend coding project from Tik Tok. In this post I'll go over my tech stack.

https://www.benjamincarlson.io/blog/tik-tok-coding-idea

**Comment by kmacoders** [comment]
test

**Comment by bjcarlson42** [comment]
works!
